expanding the default roles section (#10983)
opensearch-project/documentation-website: _security/access-control/users-roles.md

The Security plugin includes several predefined roles that serve as useful defaults.

### Built-in roles

The following table lists built-in static roles that are always provided:

| **Role** | **Description** |
| :--- | :--- |

> | `all_access`| Superuser-style role, full cluster access, all cluster operations, write to all indexes, and all tenants. |

Reviewer: ```suggestion
| `all_access`| Superuser role with full cluster access. Grants permissions to perform all cluster operations, write to all indexes, and access all tenants.|
```

| `kibana_server` | Role used by the OpenSearch Dashboards server user to read/write its internal saved objects and system indexes. Don’t assign to human users. |
| `kibana_user` | Lets a user sign in and use Dashboards. This includes cluster read and search, index monitoring, and writes to OpenSearch Dashboards indexes. Pair with read permissions to your data. |

> | `logstash`| Grants Logstash the permissions it needs to interact with OpenSearch |

Reviewer: ```suggestion
| `logstash`| Grants Logstash the permissions it needs to interact with OpenSearch. |
```

| `notifications_full_access`| Grants full permissions to perform all Notifications actions. |
| `notifications_read_access`| Grants permissions to view Notifications configuration/channels and features but not modify them. |
| `point_in_time_full_access`| Grants full permissions to perform all Point-in-Time operations.|
| `reports_instances_read_access`| Grants permissions to generate on-demand reports and download existing report instances, but not to view/create report definitions. |
| `security_analytics_ack_alerts`| Grants permissions to view and acknowledge Security Analytics alerts. |
| `security_analytics_full_access` | Grants full permissions to use all Security Analytics functionality. |
| `security_analytics_read_access` | Grants permissions to view Security Analytics detectors, alerts, findings, mappings, and rules. |
| `snapshot_management_full_access`| Grants full permissions to perform all snapshot management actions (including repositories and snapshots).|
| `snapshot_management_read_access`| Grants permissions to view snapshot management policies, repositories, and snapshots without modifying them. |

> For breakdown of individual permissions assigned to each role see [roles.yml](https://github.com/opensearch-project/security/blob/main/config/roles.yml)

Reviewer 2: ```suggestion
For a breakdown of individual permissions assigned to each role, see [roles.yml](https://github.com/opensearch-project/security/blob/main/config/roles.yml).
```

| `notifications_read_access`| Grants permissions to view Notifications configuration/channels and features but not modify them. |
| `point_in_time_full_access`| Grants full permissions to perform all Point-in-Time operations.|
| `reports_instances_read_access`| Grants permissions to generate on-demand reports and download existing report instances, but not to view/create report definitions. |
| `security_analytics_ack_alerts`| Grants permissions to view and acknowledge Security Analytics alerts. |
| `security_analytics_full_access` | Grants full permissions to use all Security Analytics functionality. |
| `security_analytics_read_access` | Grants permissions to view Security Analytics detectors, alerts, findings, mappings, and rules. |
| `snapshot_management_full_access`| Grants full permissions to perform all snapshot management actions (including repositories and snapshots).|
| `snapshot_management_read_access`| Grants permissions to view snapshot management policies, repositories, and snapshots without modifying them. |

### Detailed permissions breakdown

The following is a detailed list of permissions assigned to each demo role:

> ```json

Author: @cwperks I have updated the PR as suggested
